entrypoint.sh

```diff
@@ -44,18 +48,23 @@ main() {
     echo Building with flags: ${BUILD_FLAGS:+"$BUILD_FLAGS"}
     zola build ${BUILD_FLAGS:+"$BUILD_FLAGS"}
 
-    echo "Pushing artifacts to ${GITHUB_REPOSITORY}:$remote_branch"
+    if ! ${BUILD_ONLY}; then
```

Comment:
Would you mind switching the logic check here to not have the `not (!)` operator here and switch the resulting code paths.
It'll justify having the `BUILD_ONLY` variable name and make it a bit easier to read.

Reply:
Sure, I can make this change right now.

Also, prior to this PR, I tested this change against my own Github Pages site so I could see the results from a PR and a commit to my default branch.

PR: https://github.com/tjtelan/tjtelan.github.io/actions/runs/132888874
Default branch: https://github.com/tjtelan/tjtelan.github.io/actions/runs/132889356